### PR TITLE
Supporting several ruby versions

### LINKED ISSUE #6: Supporting multiple ruby versions
We need to check all specs for versions:
- ruby-head
- 2.4.2
- 2.4.0
- 2.3.5
- 2.3.1
- 2.2.5
- 2.2.0

### PR DESCRIPTION
**Pull request by nulldef**
resolve #6

### CODE REVIEW

**Comment by nulldef** [comment]
Waiting for #4

File: .travis.yml [hunk 1]
```diff
@@ -5,4 +5,10 @@ cache: bundler
 before_install: gem install bundler
 script: bundle exec rspec
 rvm:
+  - ruby-head
+  - 2.4.2
   - 2.4.0
+  - 2.3.5
+  - 2.3.1
```
Comment (JelF) [inline .travis.yml:12]: Is there any reason to support 2.3.1 and 2.2.0 if 2.3.5 and 2.2.5 supported?

Minor ruby version should always be updated after new release anyway

File: lib/statum/event.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 module Statum
   # Class for storing event info
   class Event
-    attr_reader :from, :to, :before, :after
+    attr_reader :from, :to
```
Comment (JelF) [inline lib/statum/event.rb:4]: Would this changes be reverted in favor of #4 ?
Single responsobility principle suggest not to add logic-specific methods to model. 

It also makes things clearer, if proc application logic is not spread

File: lib/statum/event.rb [hunk 3]
```diff
@@ -31,15 +49,27 @@ def can_fire?(current_state)
...
-    def after?
-      !after.nil?
+    def after?(instance)
+      !after(instance).nil?
+    end
+
+    private
+
+    def hook(instance, hook)
+      hook.is_a?(Symbol) ? instance.method(hook) : hook.to_proc unless hook.nil?
```
[the remark this answers is not in the document]
Comment (JelF) [reply lib/statum/event.rb:72]: This better should be memoized in event constructor. If bounding Event model to instance seems bad, see coment above

**Review by JelF — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (nulldef) [reply .travis.yml:12]: It is just test PR, and you can don't pay attension on it

**Comment by nulldef** [comment]
It was a test request, just for running travis and I gonna change all of this after #4 merging

**Comment by nulldef** [comment]
resolved in #9

**Pull request closed without merging**